Messages/Messages.tsx

```diff
@@ -101,6 +106,9 @@ const messages = {
   ${Prefix.PREFIX}${Commands.STAKE} <amount> | all - stake funds
   ${Prefix.PREFIX}${Commands.UNSTAKE} <amount> | all - unstake funds
   ${Prefix.PREFIX}${Commands.HARVEST} <amount> | all - harvest reward
+  ${Prefix.PREFIX}${Commands.BORROW} gcd with <amount> <token> with <percent>% risk | on audit
```

Comment:
We need to add [UNAUDITED] here or better yet - add these commands to an unlisted directory so that you can find them only if you know about it